test/cobalt/evaluators/relevance.py

```
@@ -0,0 +1,219 @@
+
+    Returns:
+        A float between 0.0 and 1.0 representing length appropriateness.
+    """
+    if not expected:
+        return 1.0 if not output else 0.5
+
+    ratio = len(output) / len(expected)
+
+    if 0.5 <= ratio <= 2.0:
+        return 1.0
+    elif 0.3 <= ratio < 0.5:
```

Ruff (0.15.7)

[warning] Unnecessary `elif` after `return` statement

Remove unnecessary `elif`

(RET505)